docker-for-windows/troubleshoot.md

```diff
@@ -512,19 +488,18 @@ with Linux containers.
   executed inside a VM which itself runs inside a VM. CPU utilization is also
   likely to be higher.
 
-* Ensure "PMU Virtualization" is turned off in Parallels on Macs. Check the 
-  settings in **Hardware → CPU & Memory → Advanced Settings → PMU 
+* Ensure "PMU Virtualization" is turned off in Parallels on Macs. Check the
+  settings in **Hardware → CPU & Memory → Advanced Settings → PMU
```

Comment:
Do we use arrows or > to signify a string of menu choices? For ex, File > Edit. I think > is the typical style.

Reply:
Fixed in commit 59aa7e5.